static/src/javascripts/projects/common/utils/detect.js

```diff
@@ -109,6 +109,15 @@ define([
         return totalTime;
     }
 
+    function getTimeOfDomComplete(performance) {
+        var perf = performance || window.performance || window.msPerformance || window.webkitPerformance || window.mozPerformance;
```

Comment:
Yes, we should have a conversation about some sort of standardised object to expose this data.